src/System.Private.CoreLib/shared/Interop/Unix/System.Native/Interop.GetSystemTimeAsTicks.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetSystemTimeAsTicks")]
+        internal static extern long GetSystemTimeAsTicks();
+    }
+}
```

Comment:
My preference moving forward is for files like this that already exist in corefx to be moved in a corefx PR to the right shared location, and then have it be mirrored over to coreclr/corert/mono.  I realize that adds a small amount of friction, in that there's a small delay, but it's helpful for history, and also for avoiding the duplication that we end up getting when we forget to subsequently move corefx over to the shared one, and we end up with the same file multiple times and corefx not using the same one that coreclr/corert/mono are using.

Reply:
Already done that way for this particular one.

Reply:
> Already done that way for this particular one.

Oh, ok, thanks.  I commented on it because it's showing up here as new in the diff.

Reply:
I like that you restated the general guidance because I had to learn it too. The PR has a "conflict" with the file that was moved (the contents are identical, history in CoreFX is preserved).

Reply:
I see, this PR just hasn't been rebased yet.  Thanks.